modules/primer-buttons/README.md

```diff
@@ -245,6 +245,21 @@ Use `.hidden-text-expander` to indicate and toggle hidden text.
 
 You can also make the expander appear inline by adding `.inline`.
 
+
+#### Details element summary buttons
+
+You can apply `.btn` classes to `<summary>` elements, and they the
+selected/active styles will be applied when the parent `<details>` is open.
```

Comment:
Looks like there are some typos here and could do with some tweaks. Suggest:
> You can apply the `btn` class to the `<summary>` element so that it gains the appearance of a button, and selected/active styles when the parent `<details>` is open.